### ISSUE TITLE
update https://marketplace.eclipse.org with feature id

### ISSUE THREAD

**Issue by paulvi** [nextinterfaces/http4e#3]
Please update https://marketplace.eclipse.org/content/http4e-http-and-rest-client-eclipse with feature id to enable install with MPC marketplace client

P.S. and update license as Apache/open source

**Comment by nextinterfaces** [comment]
Right. Yet, I don't seem to be able to recover my initial email I used to login Eclipse marketplace

**Comment by paulvi** [comment]
Send email to marketplace support `marketplace@e.o`

**Comment by nextinterfaces** [comment]
Sent ...

**Comment by nextinterfaces** [comment]
Sent ...

**Comment by paulvi** [comment]
up

`marketplace@eclipse.org`

**Comment by paulvi** [comment]
Or just create new Eclipse account and new marketplace entry
